src/hdusd/export/mesh.py

```diff
@@ -203,6 +203,38 @@ def sync(obj_prim, obj: bpy.types.Object, mesh: bpy.types.Mesh = None, **kwargs)
         return
 
     stage = obj_prim.GetStage()
+    parent_prim = None
+    parent_object = None
+
+    if obj.parent is not None and obj_prim.GetName() != Tf.MakeValidIdentifier(obj.original.name_full):
+        parent_object = obj.original
+        parent_prim = stage.GetPrimAtPath(f"/{Tf.MakeValidIdentifier(obj.original.name_full)}")
```

Comment:
Use sdf_name() function from export/object.py instead Tf.MakeValidIdentifier(...)
Please change it whenever it is possible